examples/community/rerender_a_video.py

```diff
@@ -307,6 +307,7 @@ def __init__(
         feature_extractor: CLIPImageProcessor,
         image_encoder=None,
         requires_safety_checker: bool = True,
+        device=None,
```

Comment:
why do we need to this?
we can just do `pipe.to()` after we create the pipeline, no?